### DISCUSSION TITLE
一款更简单的小狼毫皮肤编辑工具

### DISCUSSION THREAD

**Discussion by pdog18** [rime/home#1262] [Show and tell]
考虑到 [西米](https://bennyyip.github.io/Rime-See-Me/) 使用起来不够方便，
发布一个更简单的在线小狼毫皮肤编辑工具 [润笔](https://pdog18.github.io/rime-soak/#/theme)
希望大家提一些意见

**Comment by yangbohz** [comment]
赞一个，调色很方便。

**Comment by MeoteCarlo** [comment]
很不错，感谢。👍

**Comment by fxliang** [comment]
页面技术不熟，乱改西米如下😁

https://fxliang.github.io/RimeSeeMe/

**Reply by ghost** [reply]
如果能像搜狗那样添加图片就好了。来个搜狗主题，很有ntr的感觉

**Reply by ghost** [reply]
或者来个win11主题
